pom.xml

```diff
@@ -1761,6 +1761,9 @@
                 <!-- content of file does not matter at all -->
                 <exclude>**/RegexpOnFilenameCheckTest.class</exclude>
 
+                <!-- due to https://github.com/checkstyle/checkstyle/pull/13179 -->
```

Comment:
It not clear to me why it is excluded. Link leads to the current PR with no explanation. Could you make the reason more clear?

Reply:
Actually, we have used verify method over here to test the code.
id we remove the suppress then it will force us to use verify withInlineconfigParser method and we as this is outside of check it will not work

Reply:
I don't understand what you mean. Could you extend the comment with few words explanation please like the line just above it?

Reply:
@rdiachenko I have updated the comment please look